### DISCUSSION TITLE
R code chunks are labeled as S

### DISCUSSION THREAD

**Discussion by gonzalezbensmol** [facelessuser/pymdown-extensions#1650] [Q&A] [answered]
### Description

When highlight R code pymdown highlights the code as S. 

### Minimal Reproduction

![image](https://user-images.githubusercontent.com/93791838/160477970-f12f97ef-7ca5-4c44-80e5-4a9cb48f32bb.png)

### Version(s) & System Info

- Operating System: Github Pages
- Python Version: .3.8
- Package Version: 9.3

**Comment by facelessuser** [comment, marked as answer]
I'm assuming you are using the title feature?

The reason is due to the lexer name. It is used for other languages as well: https://github.com/pygments/pygments/blob/master/pygments/lexers/r.py#L74

Notice:

```py
    """
    For S, S-plus, and R source code.
    .. versionadded:: 0.10
    """
```

They just chose `S` as the main name.

You may want to try a title remap: https://facelessuser.github.io/pymdown-extensions/extensions/superfences/#code-block-title-headers

```py
extension_configs = {
    "pymdownx.highlight": {
        "auto_title": True,
        "auto_title_map": {
            "S": "R"
        }
    }
}
```

**Comment by gonzalezbensmol** [comment]
Yes, currently the follow is being utilized. 

  - pymdownx.highlight:
      anchor_linenums: true
      auto_title: true
      linenums: true

Where exactly do we place the extension configs? In the mkdocs.yml ?

**Comment by facelessuser** [comment]
In the same way that you enable `anchor_linenums` etc. If you post who you are doing that, I can show how you add them.

**Comment by gonzalezbensmol** [comment]
Looks as if this was the fix. Thank you for the help!!!!! Much appreciated. 

auto_title_map: {"S": "R"}

![image](https://user-images.githubusercontent.com/93791838/160482054-3690a874-f6b5-4084-b824-0e6ed5dee045.png)
